parsec/interfaces/ptg/ptg-compiler/jdf2c.c

```diff
@@ -6753,7 +6756,23 @@ static void jdf_generate_code_hook_gpu(const jdf_t *jdf,
     } else {
         weight = "1.";
     }
-    coutput("  ratio = %s;\n", weight);
+    coutput("  __ratio = %s;\n", weight);
+
+    jdf_find_property( body->properties, "optype", &optype_property );
```

Comment:
Recommend renaming this as 'precision=double',

What about 'precision=double complex'? 

Also what is triple precision?

Reply:
This comment will not apply when we implement plan below.